test/testlib/minikube_utilities.go

```diff
@@ -1287,6 +1287,30 @@ func LabelNodes(t *testing.T, namespaceName string, labelName string, labelValue
 	}
 }
 
+func LabelNodesIfMissing(t *testing.T, labelName string, labelValue string) []string {
```

Comment:
Could you please include a docstring for this function? I'm confused how the return value is consumed. It looks like the current values for different nodes are retuned, but how does the consumer know which one is from which node? Maybe you should return a `map` name to the current label value?

Reply:
It returns the list of all the current node values. You are right that returning a mapping of node name to current value would be better, because it makes some other changes I want to make easier.